lib/pages/timetable/view/timetable_page.dart

```diff
@@ -76,8 +76,9 @@ class _TimetablePageState extends State<TimetablePage> {
           onPressed: () => Navigator.of(context).push(
             MaterialPageRoute<ChangeNotifierProvider>(
               builder: (_) => ChangeNotifierProvider.value(
-                  value: Provider.of<ClassProvider>(context),
-                  child: const ClassesPage()),
+                value: Provider.of<ClassProvider>(context),
```

Comment:
What's with this change? Seems like only formatting, please revert so we don't unnecessarily mess with the history of this file.